examples/diffusion_limited_aggregation.jl

```diff
@@ -0,0 +1,123 @@
+using Agents, Random, LinearAlgebra
+Random.seed!(42)
+
+@agent Particle ContinuousAgent{2} begin
+    radius::Float64
+    is_stuck::Bool
+    twist_axis::Array{Float64,1}
+end
+
+Particle(
+    id::Int,
+    radius::Float64,
+    twist_clockwise::Bool;
+    pos = (0.0, 0.0),
+    is_stuck = false,
+) = Particle(
+    id,
+    pos,
+    (0.0, 0.0),
+    radius,
+    is_stuck,
+    [0.0, 0.0, twist_clockwise ? -1.0 : 1.0],
+)
+
+properties = Dict(
+    :speed => 0.5,   # dt multiplier
+    :wiggle => 0.55,  # position offset
+    :attraction => 0.45,  # absolute attraction value
+    :twist => 0.55,   # absolute tangential velocity
+    :clockwise_fraction => 0.0,
+    :spawn_count => 0,
+    :min_radius => 1.0,
+    :max_radius => 2.0,
+)
+
+rand_circle() = (θ = rand(0.0:0.1:359.9); (cos(θ), sin(θ)))
+particle_radius(min_radius::Float64, max_radius::Float64) = min_radius <= max_radius ?
+rand(min_radius:0.01:max_radius) : min_radius
+function agent_step!(agent::Particle, model)
+    if agent.is_stuck
+        return
+    end
+    for id in nearby_ids(agent.pos, model, agent.radius)
+        if model[id].is_stuck
+            agent.is_stuck = true
+            model.spawn_count += 1
+            return
+        end
+    end
+    radial = model.space.extent ./ 2.0 .- agent.pos
+    radial = radial ./ norm(radial)
+    tangent = Tuple(cross([radial..., 0.0], agent.twist_axis)[1:2])
+    agent.vel =
```

Comment:
Yep, I think this is better than what you had before and makes the most sense for the moment. Maybe this can be improved upon in future versions, but fine for now.